### PR TITLE
Adding new facts

### PR DESCRIPTION
**Pull request by fabriciorissetto**
Adding new facts about Chuck Norris (using spaces instead of tabs)

### CODE REVIEW

File: src/main/java/hudson/plugins/chucknorris/FactGenerator.java [hunk 1]
```diff
@@ -98,7 +98,20 @@
...
-            "Chuck Norris' preferred IDE is hexedit."};
+            "Chuck Norris' preferred IDE is hexedit.",
+            "Chuck Norris is immutable. If something's going to change, it's going to have to be the rest of the universe.",
+            "Chuck Norris' addition operator doesn't commute; it teleports to where he needs it to be.",
+            "Anonymous methods and anonymous types are really all called Chuck Norris. They just don't like to boast.",
+            "Chuck Norris doesn't have performance bottlenecks. He just makes the universe wait its turn.",
+            "Chuck Norris does not use exceptions when programming. He has not been able to identify any of his code that is not exceptional.",
+            "When Chuck Norris' code fails to compile the compiler apologises.",
+            "Chuck Norris does not use revision control software. None of his code has ever needed revision.",
+            "Chuck Norris can recite π. Backwards.",
```
Comment (batmat) [inline src/main/java/hudson/plugins/chucknorris/FactGenerator.java]: small b instead IMO

Comment (batmat) [inline src/main/java/hudson/plugins/chucknorris/FactGenerator.java]: OK, no, forget it, I just got it reading once again :)

**Comment by batmat** [comment]
Not sure I understand why you talk about spaces vs. tabs? There were already spaces anyway as I just checked?

Thanks for your work :smile: !

**Comment by batmat** [comment]
(btw, I just realized I wasn't watching this repo, hence the 10+ days reaction, fixed now.)

**Comment by batmat** [comment]
FTR, just released 1.0 with it. Enjoy! :-)

**Comment by fabriciorissetto** [comment]
Nice!!

**Pull request merged**